includes/functions.php

```diff
@@ -150,6 +150,22 @@ function wbz404_pluginActivation() {
 	) ENGINE=MyISAM " . $charset_collate . " COMMENT='404 Redirected Plugin Logs Table' AUTO_INCREMENT=1";
 	$wpdb->query($query);
 
+  $query = "CREATE TABLE IF NOT EXISTS `" . $wpdb->prefix . "wbz404_ignored_IP_addresses` (
+    `ID` bigint(40) NOT NULL auto_increment,
+    `IP_address` varchar(255) NOT NULL,
+    PRIMARY KEY  (`ID`),
+    KEY `IP_address` (`IP_address`)
+  ) ENGINE=MyISAM " . $charset_collate . " COMMENT='404 Redirected Plugin ignored IP addresses' AUTO_INCREMENT=1";
+  $wpdb->query($query);
+
+  $query = "CREATE TABLE IF NOT EXISTS `" . $wpdb->prefix . "wbz404_ignored_bots` (
+    `ID` bigint(40) NOT NULL auto_increment,
+    `user_agent` varchar(255) NOT NULL,
+    PRIMARY KEY  (`ID`),
+    KEY `user_agent` (`user_agent`)
+  ) ENGINE=MyISAM " . $charset_collate . " COMMENT='404 Redirected Plugin ignored IP bots' AUTO_INCREMENT=1";
+  $wpdb->query($query);
+
```

Comment:
Super nit-picky, but let's match the indentation of the file.  (Turn these spaces into tabs)